app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,16 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def get_rating(self) -> int:
+        return 3 * self._bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self._nickname}. "
+                f"{self._nickname} has bow of the {self._bow_level} level")
```

Comment:
Consider using double quotes for consistency, as the checklist suggests using one style of quotes, preferably double quotes.